features/facebookImport/rollup.config.js

```diff
@@ -8,6 +8,8 @@ import replace from "@rollup/plugin-replace";
 import serve from "rollup-plugin-serve";
 import svg from "rollup-plugin-svg";
 
+import genPodjs from "@polypoly-eu/podjs/bin/genPodjs.js";
```

Comment:
Great. So it's really a function :+1:

Reply:
... I was confused with the `bin` path. Do you still need it there, since you're using it as a simple function? Why not move it to a `utils` directory or something like that?

Reply:
set it as default export - so guess we're fine to have it in `bin` now? (following @fhd tastes :P)

Reply:
Well, not a script, so best practices advise us to not let it there. And it does not have any functional dependency from `podjs`, so arguably it shouldn't be there either. We do have a specific directory for the other rollup plugin we have; https://github.com/polypoly-eu/polyPod/tree/main/dev-utils/rollup-plugin-copy-watch. Putting it there (and with similar name) would probably be the best fit.

Reply:
I don't think this tiny thing is worth its own package, I think I suggested to put it in _podjs/rollup-plugin-gen-podjs_. I thought we were gonna do a script in `bin`, and provide a rollup plugin that simply calls that script -> That'd be my tastes. But as I wrote, I'm fine with having _only_ a script, or even _only_ a Rollup plugin. This is a relatively small change, it shouldn't take so much back and forth, let's just make it work I'd say.

Reply:
The problem is we're bringing in a host of dependencies where we use it we don't really need. That's going to unnecessarily add up to build times, and do so repeatedly.

Reply:
not clear if you mean `rollup-plugin-gen-podjs` as a directory or as a script. Don't like it either way; the root directory for a module should be free of non-standard files.